artificial-scrum-master/apps/artificial.scrum.master/Artificial.Scrum.Master.Infrastructure/ExternalServices/OpenAITaskSuggestionService.cs

```diff
@@ -8,22 +8,16 @@
 
 namespace Artificial.Scrum.Master.Infrastructure.ExternalServices;
 
-internal class OpenAITaskSuggestionService : ITaskSuggestionService
+internal class OpenAITaskSuggestionService(
+    IMemoryCache memoryCache,
+    IOpenAIService openAiService) : ITaskSuggestionService
 {
-    private readonly IMemoryCache _memoryCache;
-    private readonly IOpenAIService _openAIService;
+    private readonly IMemoryCache _memoryCache = memoryCache;
```

Comment:
troche niespójne to, że raz tak a raz jednak konstruktor zwykły. do posprzątania kiedyś